examples/console/main.go

```diff
@@ -83,20 +93,27 @@ func run() error {
 	escrowAccountKey := keypair.MustRandom()
 	fmt.Fprintln(os.Stdout, "escrow account:", escrowAccountKey.Address())
 
-	submitter := &Submitter{
-		HorizonClient:     horizonClient,
+	horizon := &horizon.Horizon{
+		HorizonClient: horizonClient,
+	}
+	submitter := &submit.Submitter{
+		SubmitTxer:        horizon,
```

Comment:
Should this be `SubmitTxEr`? Not sure what the convention is, but this was a bit confusing to me.

Reply:
No SubmitTxer because it is an interface for the SubmitTx function. It's a horrible name, I couldn't think of anything else in the moment but I can change it to something better. Originally I had `Submitter` but then now there are two different types with the same name. Interestingly Horizon's own submission code has this exact problem, it has two Submitter types that wrap each other and do different things, so I at least don't feel alone in this predicament.